sdk/loadtestservice/azure-analytics-loadtestservice/tests/test_smoke.py

```diff
@@ -0,0 +1,34 @@
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+from json import load
+from testcase import LoadtestserviceTest, LoadtestservicePowerShellPreparer
+
+class LoadtestserviceSmokeTest(LoadtestserviceTest):
+
+    @LoadtestservicePowerShellPreparer()
+    def test_smoke_create_or_update_test(self, loadtestservice_endpoint):
+        client = self.create_client(loadtestservice_endpoint)
+        test_id="000001" 
+        body_test={"resourceId":"/subscriptions/123456/resourceGroups/fake/providers/Microsoft.LoadTestService/loadtests/fake",
+                "testId":"000001","description":"","displayName":"test",
+                "loadTestConfig":{"engineSize":"m","engineInstances":1,"splitAllCSVs":False},"secrets":{},"environmentVariables":{},
+                "passFailCriteria":{"passFailMetrics":{}},"keyvaultReferenceIdentityType":"SystemAssigned","keyvaultReferenceIdentityId":None}
+        result = client.test.create_or_update_test(test_id, body_test)
+        assert result is not None
+
+    @LoadtestservicePowerShellPreparer()
+    def test_smoke_list_search(self, loadtestservice_endpoint):
+        client = self.create_client(loadtestservice_endpoint)
+        result = client.test.list_load_test_search()
+        assert result is not None
+
+    @LoadtestservicePowerShellPreparer()
+    def test_smoke_delete_test(self, loadtestservice_endpoint):
```

Comment:
please add a test for `upload_file_test` bc that's a handwritten method

Reply:
Duplicate at: https://github.com/Azure/azure-sdk-for-python/pull/25264#discussion_r925286925